### ISSUE TITLE
What is HEAD?

### ISSUE THREAD

**Issue by AlexDaniel** [perl6/whateverable#49]
```
10:56:12 <bartolin> committable6: 2016.09,HEAD say (my int $ = 2**32) * (my int $ = 2**63)
10:56:13 <committable6> bartolin, ¦«2016.09»: 0␤¦«HEAD»: This type cannot unbox to a native integer: P6opaque, Failure␤  in block <unit> at /tmp/5gdU3BwADu line 1␤ «exit code = 1»
```

It says `¦«HEAD»` but gives no information on how old it actually is. Should we print something like `¦«HEAD(ffeeaa22)»` ?

**Comment by AlexDaniel** [comment]
@MasterDuke17 what do you think?

**Comment by AlexDaniel** [comment]
Fixed in bbede988f697ed4b0c6598a4d471a1653678b8b2
